test/integration/targets/docker_swarm/aliases

```diff
@@ -1,5 +1,4 @@
 posix/ci/group2
```

Comment:
The group names have changed recently (yesterday or so?). I think it should be `shippable/cloud/group2`; is that correct, @mattclay?

Reply:
(Though I guess this will be fixed automatically when merging/rebasing to current devel.)